sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/PemReaderTests.Keys.cs

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using Azure.Core;
+using NUnit.Framework;
+
+namespace Azure.Security.KeyVault.Certificates.Tests
+{
+    public partial class PemReaderTests
```

Comment:
Should there be a similar comment about excluding this file from credscan?

Reply:
You're right. I clearly forgot that here. I'll submit a new PR since I don't think I have anything else KV-related planned today.